flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlTableComputedColumn.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.sql.parser.ddl;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlCharStringLiteral;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlSpecialOperator;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.ImmutableNullableList;
+
+import javax.annotation.Nullable;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Table computed column of a CREATE TABLE DDL.
+ */
+public class SqlTableComputedColumn extends SqlCall {
+	private static final SqlSpecialOperator OPERATOR =
+		new SqlSpecialOperator("COLUMN_DECL", SqlKind.COLUMN_DECL);
+
+	private SqlNode identifier;
+	private SqlNode expr;
+	private SqlCharStringLiteral comment;
+
+	public SqlTableComputedColumn(SqlNode identifier, SqlNode expr, @Nullable SqlCharStringLiteral comment, SqlParserPos pos) {
+		super(pos);
+		this.identifier = requireNonNull(identifier, "Column identifier should not be null");
+		this.expr = requireNonNull(expr, "Column expression should not be null");
+		this.comment = comment;
+	}
+
```

Comment:
Can we merge this class into `SqlTableColumn` ? Just like what we do in schema `TableColumn`.

Reply:
@danny0405 Yes, I have already merged `SqlTableComputedColumn` into `SqlTableColumn`. Please review this pull request again.